changelog.txt

```diff
@@ -1,3 +1,25 @@
+13.1 (2023-04-10)
+------------------------------------------------------------------------
+Feature: [NewGRF] Engine name callback. (#10399)
+Change: Avoid crashing to the side of a train (#10496)
+Fix 8361cf5a73: Missing bounds check for house specs. (#10625)
+Fix: Update top toolbar tooltips for added features. (#10616)
+Fix: NewGRF Action3 (+others) validation (#10601)
+Fix: Clicking on editbox clear button didn't take account of padding. (#10583)
+Fix: [Script] Access to enum/consts defined outside of main.nut (#10573)
+Fix #10568: "can savegame be loaded check" failed in dedicated server (#10572)
+Fix #10554: Let Scrollbar::SetPosition clamp instead of assert. (#10555)
+Fix: Network server highlight invisible with RTL layout. (#10551)
+Fix: Restore using founder client name as company manager name (#10535)
+Fix: Don't send unused tile field over the network (#10507)
+Fix #10477: Not enough space for text due to rounding down (OSX) (#10489)
+Fix #10486: [Script] Debug window requires AIs to be started before GS (#10487)
+Fix #10469, 5e14a20: [Script] League Table rating element is a int64 everywhere else (#10471)
+Fix #10465: Delay closing of network join progress window (#10466)
+Fix #10461, Fix #10280: Town actions GUI issues (#10462)
+Fix #10059: Clamp config item values to int, disallow negative random deviation, and allow values up to 12 digits including '-' (#10444)
```

Comment:
Fix #10059: Script config values stored in the config file could cause crashes (#10462)